doc/introduction/circuits.rst

```diff
@@ -171,6 +175,50 @@ for accessing the results of your program execution:
     will need to increase the cutoff dimension.
 
 
+Symbolic parameters
+-------------------
+
+The quantum operations can take both numerical and symbolic parameters. The latter fall into two types:
+
+* **Measured parameters**: Certain quantum programs require that
+  operations can be conditioned on measurement results obtained during the execution of the
+  program. In this case the parameter value is not known until the measurement is made
+  (or simulated). The latest measurement result of qumode ``i`` is referred to as ``q[i].par``.
```

Comment:
Is referred the right word here? 'denoted', or even 'available via ``q[i].par``'?

Reply:
Available might be the best choice

Reply:
```suggestion
  (or simulated). The latest measurement result of qumode ``i`` is available via ``q[i].par``.
```